public/index.html

```diff
@@ -48,7 +48,9 @@
       You need to enable JavaScript to run this app.
     </noscript>
     <div id="root">
-      <form-builder fbms-base-url="http://localhost:8091" fbms-form-fname="communication-preferences" oidc-url="http://localhost:8080/uPortal/api/v5-1/userinfo"></form-builder>
+      <form-builder fbms-base-url="http://localhost:8091" fbms-form-fname="communication-preferences" oidc-url="http://localhost:8080/uPortal/api/v5-1/userinfo">
+        <div>This is a child passed into form-builder</div>
```

Comment:
When we're close to merging, change this text to more of a real-world example (I suggest).